src/RadioButton.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { color } from 'styled-system'
+import Icon from './Icon'
+import Text from './Text'
+
+const RadioButton = ({
+  className,
+  theme,
+  onClick,
+  isSelected,
+  disabled,
+  radioButtonText
+}) => {
+  const radioIconName = isSelected ? 'radioFilled' : 'radio'
+  const radioIconColor = isSelected ? 'blue' : 'borderGray'
+
+  function handleClick() {
+    if (disabled || isSelected) return
+    onClick()
+  }
+
+  return (
+    <RadioButtonWrap
+      className={className}
+      onClick={handleClick}
+      isSelected={isSelected}
+      disabled={disabled}
+      theme={theme}
+    >
+      <RadioIconWrap>
+        <Icon name={radioIconName} size={24} color={radioIconColor} />
+      </RadioIconWrap>
+      <Text.span pl={4}>{radioButtonText}</Text.span>
```

Comment:
I think we'll want to have a low-level primitive for `<input type='radio'>` for composition. Ideally we'd have a Radio component that can be nested inside the Label – e.g.

```jsx
<Label>
  <Radio />
  Text for the radio label as children
</Label>
```
```